addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -1,9 +1,14 @@
 
 @each $font-name, $font-config in $o-theme-font-configs {
     $url: map-get($font-config, 'url');
+    $attachment: map-get($font-config, 'attachment');
+    $name: map-get($font-config, 'name');
     @if $url {
         @import url("https://fonts.googleapis.com/css?family=#{unquote($url)}&display=swap");
     }
+    @if $attachment {
+        @import url("/web/content/#{$attachment}/google-font-#{unquote($name)}");
+    }
```

Comment:
Same, maybe we could create a function `$font-config -> @import` to share this code (would require to also change `o-get-font-info` to return the whole font config if `$config-key` is null for example, small optimizations, I let you decide).